README.md

```diff
@@ -106,6 +106,36 @@ A prebuilt Docker image is provided as well:
 docker run -v $(pwd)/nitter.conf:/src/nitter.conf -d -p 8080:8080 zedeus/nitter:latest
 ```
 
+To run the prebuilt Docker image via docker-compose:
+```yaml
+version: '3.5'
+services:
+  nitter:
+    volumes:
+        - ./nitter.conf:/src/nitter.conf
+    ports:
+        - 8080:8080
+    image: zedeus/nitter
+    container_name: nitter
```

Comment:
```suggestion
    container_name: nitter
    restart: unless-stopped
```

The redis service is already set to restart `unless-stopped`. I think it make sense to do the same for the nitter service.